Fixed attachment name decoding bug in MS graph single user (#23982)
demisto/content: Packs/MicrosoftGraphListener/ReleaseNotes/1_3_6.md

#### Integrations
##### Microsoft Graph Mail Single User

> - Fixed an issue in ***fetch-incidents*** where an attachment file name didn't was decoded correctly.

Reviewer: ```suggestion
- Fixed an issue in ***fetch-incidents*** command where attachment file names with special characters were not decoded correctly.
```
